### PR TITLE
add wordpress container

### PR DESCRIPTION
**Pull request by sophia0328**
Signed-off-by: Gong Sophia <sophia.gong@intel.com>

### CODE REVIEW

File: wordpress/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	} | tee php-fpm.d/docker.conf \
+	&& { \
+		echo '[global]'; \
+		echo 'daemonize = no'; \
+		echo; \
+		echo '[www]'; \
+		echo 'listen = 9000'; \
+	} | tee php-fpm.d/zz-docker.conf \
+	&& { \
+		useradd -U www-data; \
```
Comment (qzheng527) [inline wordpress/Dockerfile]: Is it really necessary to change the ownership of the "/var/www/html"?

**Comment by sophia0328** [comment]
@qzheng527， thanks for the review.  
1. i've tried to refine docker-entrypoint.sh, we don't need apache2 command item. For other command lines that aims to create/configure wp-config.php are necessary. it seems the default wordpress configure file is wp-config-sample.php, the front part of  docker-entrypoint.sh is to create new wp-config.php from wp-config-sample.php.

2. www-data is the common account for wordpress. compared with root, this account could help to avoid many security access issue. we'd better keep it.

i will re-submit the modified wordpress ms.

**Comment by sophia0328** [comment]
new wordpress ms commit at https://github.com/clearlinux/dockerfiles/pull/169. please help to review

**Comment by sophia0328** [comment]
update this commit at https://github.com/clearlinux/dockerfiles/pull/169

**Pull request closed without merging**